### DISCUSSION TITLE
值得买网站运行错误呢

### DISCUSSION THREAD

**Discussion by asnon** [qd-today/qd#578] [General]
提示2024-12-21 19:16:38
日志：Failed at 2/27 request,
Fail assert: {"re": "error_code\":\"4", "from": "content"} from success_asserts,
Response Error : HTTP 404: Not Found,
Request URL: https://api.smzdm.com/v1/user/lottery/draw
其他网站都没问题呢，大家有遇到过么？

**Comment by ckx000** [comment]
issues提交地方错了 给你移动下。。记得补充下其他数据

**Comment by ETboxer** [comment]
同样的问题，而且公共模板也很久刷不出了，作者放弃了吗

**Comment by a76yyyy** [comment]
参考 https://github.com/qd-today/templates/issues/20#issuecomment-2440384269
